omod/src/main/webapp/resources/js/ugandaemr.js

```diff
@@ -839,3 +839,23 @@ function getLatestEncounterFromEncounterList(encounterList) {
     return latestEncounter
 }
 
+/**
+ *
+ * function returns the name of the Encounter type Uuid passed
+ * @param encounterType Uuid
+ * @returns {string} Naame of Encounters
+ */
+function getPatientTransferEncounterTypes(encounterType) {
```

Comment:
@dbaluku Why do we need the encounterTYpe name, can't we load it via Velocity and store it in a local variable?

Reply:
@ssmusoke lets go with this implementation for now